apps/web/screens/Search/Search.tsx

```diff
@@ -194,15 +195,33 @@ const Search: Screen<CategoryProps> = ({
       labels.push(item.organization[0].title)
     }
 
+    if (item.parent) {
+      if (item.parent.group) {
+        labels.push(item.parent.group.title)
+      }
+
+      if (item.parent.organization?.length) {
+        labels.push(item.parent.organization[0].title)
+      }
+    }
+
     return labels
   }
 
   const searchResultsItems = (searchResults.items as Array<
-    Article & LifeEventPage & AboutPage & News & AdgerdirPage
+    Article & LifeEventPage & AboutPage & News & AdgerdirPage & SubArticle
   >).map((item) => ({
     title: item.title,
-    description: item.intro ?? item.seoDescription ?? item.description,
-    link: linkResolver(typenameResolver(item.__typename), [item.slug]),
+    parentTitle: item.parent?.title,
+    description:
+      item.intro ??
+      item.seoDescription ??
+      item.description ??
+      item.parent.intro,
+    link:
+      item.slug.split('/').length > 1
+        ? linkResolver(typenameResolver(item.__typename), item.slug.split('/'))
+        : linkResolver(typenameResolver(item.__typename), [item.slug]),
```

Comment:
The check here might not be needed, split would result in an array with the slug even if slash is missing:
```suggestion
    link: linkResolver(typenameResolver(item.__typename), item.slug.split('/'))
```

Reply:
Oh, you are right. Nice catch :+1: